.github/workflows/test-with-edm.yml

```diff
@@ -8,7 +8,7 @@
 
 name: Test with EDM
 
-on: [pull_request, workflow_dispatch]
```

Comment:
Was this change intentional?

Reply:
Yes.  No point in running these until EDM has Pyface 8.